client-v2/src/selectors/frontsSelectors.ts

```diff
@@ -20,6 +20,8 @@ interface FrontsByPriority {
   [id: string]: FrontConfig[];
 }
 
+const selectEditorFrontIds = (state: State) => state.editor.frontIds;
```

Comment:
Would it be better if these selectors hung out with the frontsUIBundle? They relate to the UI state, not the fronts config state.

Reply:
Because frontsUIBundle depends on `getFronts` , we can't depend on anything from that file in frontsSelectors, so if `selectEditorFrontIds` is moved, `getFrontsWithPriority` should go as well.

Reply:
Sorry, I've just seen in your implementation notes -- `Probably worth looking into looking at how we split selectors in these two files in more detail.` 👍 let's do that, tools like [madge](https://github.com/pahen/madge) might help!